lib/diagnostic.rb

```diff
@@ -28,7 +30,17 @@ def example
 # responds successfully and lists all examples.
 
 RSpec.describe 'Examples API' do
-  # your test(s) here
+  def comments
+    Comment.all
+  end
+
+  describe 'GET /examples' do
+    it 'lists all examples' do
+      get '/examples'
+
+      expect(response).to be_success
```

Comment:
Want to make sure we also check that the examples coming back from the successful response are what we want with one or both of:

```
examples_response = JSON.parse(response.body)
expect(examples_response.length).to eq(examples.count)
expect(examples_response['examples'].first['title']).to eq(example[:title])
```